addons/mail/static/src/messaging/component/chat_window/chat_window.js

```diff
@@ -26,6 +26,10 @@ class ChatWindow extends Component {
                 localeTextDirection: this.env.messaging.locale.textDirection,
             };
         });
+        /**
+         * Useful to prevent click on header from wrongly focusing the window.
+         */
+        this._chatWindowHeaderRef = useRef('chatWindowHeader');
```

Comment:
would rename ref `header` (same naming as the part of BEM).

Reply:
ok